app/models/manageiq/providers/kubevirt/infra_manager.rb

```diff
@@ -181,7 +193,7 @@ def virtualization_endpoint
   end
 
   def default_authentication_type
-    ENDPOINT_ROLE
+    DEFAULT_AUTH_TYPE
```

Comment:
Hm if this constant isn't used anywhere else wonder if we could just drop it and use `:kubevirt` here (follow-up)